package.json

```diff
@@ -36,27 +36,41 @@
     "react-hamburger-menu": "^1.1.1",
     "react-native-web": "^0.12.0",
     "react-redux": "^7.1.3",
-    "react-scripts": "^1.1.5",
+    "react-scripts": "^3.3.0",
```

Comment:
Unfortunately upgrading `react-scripts` breaks the build on mobile. See #308.

Reply:
moved it back to 1.1.5